src/entity/notifications/NotificationPreferenceComment.ts

```diff
@@ -0,0 +1,13 @@
+import { ChildEntity, ManyToOne, PrimaryColumn } from 'typeorm';
+import { NotificationPreference } from './NotificationPreference';
+import { NotificationPreferenceType } from '../../notifications/common';
+import { Comment } from '../Comment';
+
+@ChildEntity(NotificationPreferenceType.Comment)
+export class NotificationPreferenceComment extends NotificationPreference {
+  @PrimaryColumn({ type: 'text', default: null })
+  commentId?: string;
```

Comment:
This should not be optional in case it's a `NotificationPreferenceComment`. We must have a commentId for that